media-gfx/oxipng/files/oxipng-9.1.3-add-system-libdeflate-feature.patch

```diff
@@ -0,0 +1,47 @@
+From 161b184373092742e9867a8b8cabad92a0395f15 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to the upstream source/reference for this patch to the top of the file.

Reply:
Done!
I didn't know that the leading block can contain (almost) free-form text...